br_stock_account/models/account_invoice.py

```diff
@@ -9,7 +9,17 @@
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
 
-    pos_fiscal = fields.Selection(related='fiscal_position_id.fiscal_type')
+    fiscal_position_type = fields.Selection(
+        related='fiscal_position_id.fiscal_type', readonly=True)
+
+    @api.multi
+    def copy(self, default=None):
+        new_acc_inv = super(AccountInvoice, self).copy(default)
+        if self.fiscal_position_type == 'import':
+            for i in range(len(new_acc_inv.invoice_line_ids)):
```

Comment:
for item in new_acc_inv.invoice_line_ids:

Reply:
Usei o range pra poder iterar no new_acc_inv e no account_invoice com a mesma variável.